eve/main.yml

```diff
@@ -9,16 +9,37 @@ branches:
 
 
 models:
-  - SetProperty: &docker_image_name
-      name: Set docker image name property
-      property: docker_image_name
-      value:
-        "%(secret:private_registry_url)s/zenko/backbeat:\
-        %(prop:commit_short_revision)s"
+  - SetProperty: &registry_url
+      name: registry
+      property: registry_url
+      value: registry.scality.com/zenko-dev/backbeat
```

Comment:
Is there a reason you chose `zenko-dev` as the registry name and not just `zenko`?

Reply:
https://registry.scality.com/harbor/projects/7/repositories I think it's because that's how we have it set up here?

Reply:
Well, my idea/state of mind, for now, is that, we're still playing with harbor. The images that we're publishing in our post-merge are for dev purposes. Therefore I don't want to give the idea of "this is production ready, please use it".

At some point we will publish the tagged and released images on harbor, and those will be on the registry.scality.com/zenko 

This is just an idea, and of course I'm open to any suggestion.

Reply:
It works for now. I had the idea that we can have nightlies/weeklies (tagged appropriately with the label nightly/weekly) in the same repo. Something like
`registry.scality.com/zenko/backbeat/nightly/nightly-20191211`, `registry.scality.com/zenko/backbeat/weekly/weekly-20191211` and `registry.scality.com/zenko/backbeat/ga/8.11.2` etc.